test/unit/ngsi-ld/examples/contextRequests/createMinimumProvisionedDevice.json

```
@@ -3,7 +3,7 @@
         "@context": "http://context.json-ld",
         "attr_name": {
             "type": "Property",
-            "value": " "
+            "value": null
```

If I have understand correctly https://github.com/telefonicaid/iotagent-node-lib/pull/946#issuecomment-737325330 this
```suggestion
            "value": { "@type": "Intangible", "@value": null }
```

should fix the problem we have with tests.

@jason-fox is my understanding correct, please?

---

Several examples of passing a `null` can be found in the [expressionBasedTransformations-test.js](https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/unit/ngsi-ld/expressions/expressionBasedTransformations-test.js) - for example: [updateContextExpressionPlugin5.json](https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/unit/ngsi-ld/examples/contextRequests/updateContextExpressionPlugin5.json#L6-L9)

In your case I would use [updateProvisionDeviceStatic.json](https://github.com/telefonicaid/iotagent-node-lib/blob/master/test/unit/ngsi-ld/examples/contextRequests/updateProvisionDeviceStatic.json#L22-L28) as a basis.